vaadin-spreadsheet-flow-parent/vaadin-spreadsheet-flow-integration-tests/frontend/custom.css

```diff
@@ -0,0 +1,62 @@
+vaadin-spreadsheet.custom::part(popup-button) {
+    background: #faeada;
+    color: brown;
+    font-weight: 600;
+}
+
+vaadin-spreadsheet.custom::part(popup-button):hover {
+    background: #f2d1c7;
+}
+
+vaadin-spreadsheet.custom::part(popup-button):after {
+    content: '▼';
+}
+
+vaadin-spreadsheet.custom::part(row-header) {
+    background-color: #f0f8ff;
+    border-right: 2px solid #add8e6;
+}
+
+vaadin-spreadsheet.custom::part(column-header) {
+    background-color: #f0f8ff;
+    border-bottom: 2px solid #add8e6;
+}
+
+vaadin-spreadsheet.custom::part(column-header selected-header) {
```

Comment:
The "selected-header" part name seems to be set incorrectly to the column headers

https://github.com/user-attachments/assets/d5e0e7bf-586a-4545-ad1d-38e71569f0cb

Reply:
> Maybe this is the intended behavior, that it’s both a column and a row header?

Hmm...not sure what you mean. The bug starts to occur at 6 seconds into the video. The custom bottom border gets applied to random column header cells as you scroll horizontally.